lib/routes/gs-developer/blog.ts

```diff
@@ -0,0 +1,64 @@
+import { Route, Data } from '@/types';
```

Comment:
Please use the [namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) `gs` instead of `gs-developer`.